modules/serverless-installing-cli-linux.adoc

```diff
@@ -25,13 +25,16 @@ $ kn: No such file or directory
 .Procedure
 
 . Download the relevant Knative (`kn`) CLI `tar.gz` archive:
-
++
+--
 * link:https://mirror.openshift.com/pub/openshift-v4/clients/serverless/latest/kn-linux-amd64.tar.gz[Linux (x86_64, amd64)]
 
 * link:https://mirror.openshift.com/pub/openshift-v4/clients/serverless/latest/kn-linux-s390x.tar.gz[Linux on IBM Z and LinuxONE (s390x)]
 
 * link:https://mirror.openshift.com/pub/openshift-v4/clients/serverless/latest/kn-linux-ppc64le.tar.gz[Linux on IBM Power (ppc64le)]
-
+--
++
+You can also download `kn` older than the latest version. For that, navigate to the corresponding directory in the link:https://mirror.openshift.com/pub/openshift-v4/clients/serverless/[Serverless client download mirror].
```

Comment:
Curious why you talk about "older than the latest version" releases. Perhaps, I'd make this more general and explain that users can download "any" version of the kn client outside of their OpenShift installation, inclusing the version for Serverless 1.27 which is supposed to work on OCP 4.12 (this whole issue is related to OCP 4.12 where kn might not be available in the OCP UI).

Reply:
@mgencur Your version specification is better, implemented. https://github.com/openshift/openshift-docs/pull/55192/commits/ec6295a92801d4b65ed29e9f0a143941872d1846

I also think we should talk about "you can download the 1.27 as well, while you can't in the Web UI" explicitly. However, how about we add this info as a release note?

Reply:
This will be true for 4.12 and any future release so it would be good to have it in the main docs. Not release notes.